linux_mode/raw2dmp/Makefile

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
This change will also be gone once the `raw2dump`'s logic is in the Python script.